index.html

```diff
@@ -128,54 +132,94 @@ <h3>Wootbit Software Develorper Agency</h3>
-            <a href="#">Get My Resume</a>  
-            <img id="bottom-left-img" src="./assets/svg/2-aboutme-mobile.svg" alt="">
-        </div>
-        <ul class="about-me-second">
-            <li class="second-card">
-                <div>
-                    <img src="./assets/png/icon-language.png" alt="">
-                </div>
-                <h3>Languages</h3>
-                <ul>
-                    <li>JavaScript</li>
-                    <li>PHP</li>
-                    <li>Go</li>
-                    <li>Ruby</li>
-                </ul>
-            </li>
-            <li class="second-card">
-                <div>
-                    <img src="./assets/png/icon-frameworks.png" alt="">
-                </div>
-                <h3>Frameworks</h3>
+        <section id="about-me">
+            <div class="about-me-first">
+                <img id="top-right-img" src="./assets/svg/1-aboutme-mobile.svg" alt="">
+                <h2>About me</h2>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+                <a href="#">Get My Resume</a>
+                <img id="bottom-left-img" src="./assets/svg/2-aboutme-mobile.svg" alt="">
+            </div>
+            <ul class="about-me-second">
+                <li class="second-card">
+                    <div>
+                        <img src="./assets/png/icon-language.png" alt="">
+                    </div>
+                    <h3>Languages</h3>
+                    <ul>
+                        <li>JavaScript</li>
+                        <li>PHP</li>
+                        <li>Go</li>
+                        <li>Ruby</li>
+                    </ul>
+                </li>
+                <li class="second-card">
+                    <div>
+                        <img src="./assets/png/icon-frameworks.png" alt="">
+                    </div>
+                    <h3>Frameworks</h3>
+                    <ul>
+                        <li>TailwindCSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Vue</li>
+                        <li>Nuxt</li>
+                    </ul>
+                </li>
+                <li class="second-card">
+                    <div>
+                        <img src="./assets/png/icon-skills.png" alt="">
+                    </div>
+                    <h3>Skills</h3>
+                    <ul>
+                        <li>Github</li>
+                        <li>Terminal</li>
+                        <li>codepen</li>
+                        <li>Figma</li>
+                    </ul>
+                </li>
+            </ul>
+        </section>
+
+        <section class="form-section">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
```

Comment:
- [ ] Kindly adjust the spacing and font-size of this text.
- Your design
![FireShot Capture 106 - Abel's Amazing Work - raw githack com](https://user-images.githubusercontent.com/46851023/134257245-b6a13e51-ccad-4b4b-9823-c988fb2ff3b6.png)


- Figma design

![Screenshot (278)](https://user-images.githubusercontent.com/46851023/134257320-7631d62d-3e48-4305-8097-dd18833e2a77.png)

Reply:
Thanks for your help